learning/courses/quantum-computing-in-practice/utility-scale-qaoa.ipynb

```diff
@@ -140,19 +150,19 @@
+    "Each iteration of these operators is called a layer. These layers can be seen as discretization of the time evolution of the system, as previously described. The alternating pattern comes from the trotter decomposition and approximates the exponential functions of non-commuting matrices. In general, the more layers or steps we include, the closer we will be to continuous time evolution, like in QAA, so in theory, the more accurate the result will be. But for this example, we'll begin by just sampling with one layer. Remember, both the cost function Hamiltonian and the mixer are parameterized, we still need to come up with optimal values for $\\gamma$ and $\\beta.$\n",
     "\n",
     "\n",
-    "### 2.\tOptimize\n",
+    "### Optimize\n",
     "\n",
     "While the circuit we just created looks pretty simple and is useful to build up an intuitive understanding, remember, the quantum chip doesn't understand what the QAOA gate is. We need to turn this into a series of single- and two-qubit “native” gates that can be performed directly on the hardware. Native gates are those that can be performed directly on the qubits. Such circuits are said to be written in the backend's Instruction Set Architecture (ISA).\n",
     "\n",
     "The Qiskit library offers a series of transpilation passes that cater to a wide range of circuit transformations. We want to make sure that the circuit is optimized for our purpose.\n",
     "\n",
     "Recall from our previous lesson that the transpilation process involves several steps:\n",
-    "•\tInitial mapping of the qubits in the circuit (i.e. decision variables) to physical qubits on the device.\n",
-    "•\tUnrolling of the instructions in the quantum circuit to the hardware native instructions that the backend understands.\n",
-    "•\tRouting of any qubits in the circuit that interact to physical qubits that are adjacent with one another.\n",
+    "- Initial mapping of the qubits in the circuit (i.e. decision variables) to physical qubits on the device.\n",
+    "- Unrolling of the instructions in the quantum circuit to the hardware native instructions that the backend understands.\n",
+    "- Routing of any qubits in the circuit that interact to physical qubits that are adjacent with one another.\n",
     "\n",
     "And as always, more details about this can be found on the documentation site.\n",
```

Comment:
Let's link to the docs directly here - here maybe? [/docs/guides/transpile](https://quantum.cloud.ibm.com/docs/en/guides/transpile) - or the staging pg?